src/commands/image/imageSource/ContainerAppUpdateStep.ts

```diff
@@ -51,13 +53,36 @@ export class ContainerAppUpdateStep<T extends ImageSourceContext & IngressContex
             name: getContainerNameForImage(nonNullProp(context, 'image')),
         });
 
-        const updating = localize('updatingContainerApp', 'Updating container app...', containerApp.name);
-        progress.report({ message: updating });
+        // Related: https://github.com/microsoft/vscode-azurecontainerapps/pull/805
+        const retries = 4;
+        await retry(
+            async (currentAttempt: number): Promise<void> => {
+                if (currentAttempt === 2) {
```

Comment:
Worth checking the error code instead of just automatically retrying and outputting this message? If it really is a propagation issue, I kind of lean towards not telling the user and just retry in the background a few times.

Reply:
Will reach out to discuss offline